sessionctx/variable/session.go

```diff
@@ -97,40 +97,40 @@ func (tc *TransactionContext) UpdateDeltaForTable(tableID int64, delta int64, co
 
 // SessionVars is to handle user-defined or global variables in current session.
 type SessionVars struct {
-	// user-defined variables
+	// Users variables
 	Users map[string]string
-	// system variables
+	// Systems variables
 	Systems map[string]string
-	// prepared statement
+	// PreparedStmts is prepared statement
 	PreparedStmts        map[uint32]interface{}
 	PreparedStmtNameToID map[string]uint32
-	// prepared statement auto increment id
+	// preparedStmtID is prepared statement of auto-incremented id
```

Comment:
s/preparedStmtID is prepared statement of auto-incremented id/preparedStmtID is id of prepared statement/